modules/monitoring-setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects.adoc

```diff
@@ -3,10 +3,16 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="setting-scrape-sample-and-label-limits-for-user-defined-projects_{context}"]
-= Setting scrape sample and label limits for user-defined projects
+[id="setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects_{context}"]
+= Setting scrape and evaluation intervals and enforced limits for user-defined projects
 
-You can limit the number of samples that can be accepted per target scrape in user-defined projects. You can also limit the number of scraped labels, the length of label names, and the length of label values.
+You can set the following scrape and label limits for user-defined projects:
+
+* Limit the number of samples that can be accepted per target scrape
```

Comment:
I'm not sure `Limit` is necessary in the bullet points. It seems implied by `the following scrape and label limits`.

Reply:
Good point. However, I think without the "limit" at the beginning it reads a little awkwardly. I was trying to find a solution here, maybe changing the introductory sentence (something like "You can set the enforced  limits  to limit the following..." repeats limit as well, or somethingl like "You can limit the following..." but then we would omit how that is done. So I think this solution seems the cleanest to me. But let me know if there are any other suggestions, thank you!